### ISSUE TITLE
Readthedoc document building fail

### ISSUE THREAD

**Issue by TheaperDeng** [intel-analytics/BigDL#4321]
https://readthedocs.org/projects/bigdl/builds/

**Comment by jason-dai** [comment]
@helenlly please take a look

```
Processing /home/docs/checkouts/readthedocs.org/user_builds/bigdl/checkouts/latest
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      error: Multiple top-level packages discovered in a flat-layout: ['apps', 'ppml', 'scala', 'docker'].
      
      To avoid accidental inclusion of unwanted files or directories,
      setuptools will not proceed with this build.
      
      If you are trying to create a single distribution with multiple packages
      on purpose, you should not rely on automatic discovery.
      Instead, consider the following options:
      
      1. set up custom discovery (`find` directive with `include` or `exclude`)
      2. use a `src-layout`
      3. explicitly set `py_modules` or `packages` with a list of names
      
      To find more information, look for "package discovery" on setuptools docs.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by pinggao187** [comment]
fix: [#4322](https://github.com/intel-analytics/BigDL/pull/4322)
pass
https://readthedocs.org/projects/bigdl/builds/16500175/

**Comment by sgwhat** [comment]
> fix: [#4322](https://github.com/intel-analytics/BigDL/pull/4322) pass https://readthedocs.org/projects/bigdl/builds/16500175/

Still happening.

**Comment by sgwhat** [comment]
> > fix: [#4322](https://github.com/intel-analytics/BigDL/pull/4322) pass https://readthedocs.org/projects/bigdl/builds/16500175/
> 
> Still failing. @pinggao187

pass.
